pkgs/development/tools/misc/bashdb/default.nix

```diff
@@ -1,10 +1,15 @@
 { stdenv, fetchurl }:
 
+let
+  basename = "bashdb-4.4-0.92";
```

Comment:
If you make the attrset passed to `mkDerivation` recursive, you can inline this, as in
```
stdenv.mkDerivation rec {
  name = "...";
   src = {
     url = ".../${name}.tar.bz2";
   }
}
```